DRPC test code changes :

jmishrarh opened the pull request:
Added test code to execute a DRPolicy is only deleted post all DRPCs are deleted. Thanks Shyam for unblocking.  
Signed-off-by: Jolly Mishra <jmishra@redhat.com>

ShyamsundarR reviewed (commented):
LGTM other than nits, but possibly requires a rebase to main

ShyamsundarR commented:
@jmishrarh request a rebase to main, due to conflict (the sprintf is already fixed in main)

jmishrarh commented:
Found multiple conflicts while rebasing/cherry-picking   drplacementcontrol_controller_test.go . Created a new branch and raised a pull request and addressed review comments. The pull request number is 465. Hence closing this pull request.
